tests/test_Restart.cpp

```diff
@@ -112,59 +112,14 @@ namespace {
     {
         return std::get<1>(vec);
     }
-} // Anonymous namespace
-
-namespace Opm {
-namespace data {
-
-// Some test specific equivalence definitions and pretty-printing. Not fit
-// as a general purpose implementation, but does its job for testing and
-// pretty-pringing for debugging purposes.
-
-std::ostream& operator<<( std::ostream& stream, const Rates& r ) {
```

Comment:
These operators might be good to have somewhere for debugging, but I guess they are not used here. A little surprised though, I would think some tests would print objects like these upon failure?

Reply:
> These operators might be good to have somewhere for debugging,

Maybe, but then they must be significantly amended to account for a lot of changes that have happened since they were introduced in commit b43461cb49 (originally OPM/opm-output@084a233, PR OPM/opm-output#98) and never maintained.

> I guess they are not used here.

Correct, they are not.

> A little surprised though, I would think some tests would print objects like these upon failure?

That depends on how we write the test statements.  Macros like `BOOST_CHECK_EQUAL` or `BOOST_CHECK_LT` require objects that can be converted to a character representation through `operator<<()`, but the more low-level test macros like `BOOST_CHECK` don't impost such requirements.